odom_frame: odom
base_frame: base_link
front_gap: 20.0
back_gap: 0.35
turn_gap: 0.35
adjust_angle: 0.05

